src/CodeGen_PyTorch.legacy.cpp

```diff
@@ -0,0 +1,480 @@
+#include <iostream>
```

Comment:
What is the purpose of this file?

Reply:
removed, this was the interface adapted to PyTorch with version < 1.0, which was pure C, not C++